WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductRowViewModel.swift

```diff
@@ -324,6 +349,24 @@ final class ProductRowViewModel: ObservableObject, Identifiable {
         }
     }
 
+    /// Returns a text-based stock quantity if there's stock, or a fall-back when stock quantity doesn't apply
+    ///
+    private func createStockQuantityText() -> String {
+        switch stockStatus {
+        case .inStock:
+            if let stockQuantity = stockQuantity, manageStock {
+                let localizedStockQuantity = NumberFormatter.localizedString(from: stockQuantity as NSDecimalNumber, number: .decimal)
+                return String.localizedStringWithFormat(Localization.stockFormat, localizedStockQuantity)
+            } else {
+                return "In Stock"
+            }
+        case .outOfStock:
+            return "Out of Stock"
+        default:
+            return ""
+        }
```

Comment:
Perhaps using the `description`, which is implemented on the model, would be better here?

Reply:
Yes! 94c3357a538f77872af2ca7473d9d0885c24fa10